datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validate_snmp_profiles.py

```diff
@@ -1,21 +1,75 @@
 import click
+import glob
+import os
 
 from .....fs import dir_exists, file_exists
-from ...console import CONTEXT_SETTINGS, abort, echo_failure
+from ...console import CONTEXT_SETTINGS, abort, echo_failure, echo_success, echo_warning, echo_info
 from . import validators
 
+from .validators.utils import (
+    initialize_path,
+    exist_profile_in_path
+)
+
+MESSAGE_METHODS = {'success': echo_success, 'warning': echo_warning, 'failure': echo_failure, 'info': echo_info}
 
 @click.command("validate-profile", short_help="Validate SNMP profiles", context_settings=CONTEXT_SETTINGS)
 @click.option('-f', '--file', help="Path to a profile file to validate")
 @click.option('-d', '--directory', help="Path to a directory of profiles to validate")
 @click.option('-v', '--verbose', help="Increase verbosity of error messages", is_flag=True)
 def validate_profile(file, directory, verbose):
+    path = initialize_path(directory)
+
     if file:
-        if not file_exists(file):
-            echo_failure("Profile file not found, or could not be read: " + file)
-            abort()
-    if directory:
-        if not dir_exists(directory):
-            echo_failure("Directory not found, or could not be read: " + directory)
-            abort()
-    validators.validate_profile(file, directory, verbose)
+        _validate_single_profile(file,path,verbose)
+
+    elif directory:
+        all_profiles_directory = get_all_profiles_directory(directory)
+        for profile in all_profiles_directory:
+            echo_info("Start validation of profile {profile}:".format(profile=profile))
+            _validate_single_profile(profile,path,verbose)
+    else:
+        echo_failure("Provide a file or directory to validate profile")
```

Comment:
Before this change, I think we validate by default `snmp/datadog_checks/snmp/data/profiles`:

https://github.com/DataDog/integrations-core/blob/c25dc14e73a066d2040f8e08ed9c84c38a2e8cc5/datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validators/schema_validation.py#L82-L86

when running `ddev meta snmp validate-profil`

But now, I'm getting:

```
§ ddev meta snmp validate-profile
Provide a file or directory to validate profile
```

I think we need to keep the old behaviour if no argument is provided. Note that, the script is run without argument on CI here: https://github.com/DataDog/integrations-core/blob/master/.azure-pipelines/templates/run-validations.yml#L55-L58

Reply:
Yes! it is a good idea!
Changed